meson_options.txt

```diff
@@ -14,3 +14,4 @@ option('tests', type: 'feature', value: 'auto', description: 'Run tests')
 option('mangoplot', type: 'feature', value: 'enabled')
 option('dynamic_string_tokens', type: 'boolean', value: true, description: 'Use dynamic string tokens in LD_PRELOAD')
 option('with_fex', type : 'boolean', value : false)
+option('with_obs', type : 'feature', value : 'disabled')
```

Comment:
it can probably be set to enabled by default. But we should add the obs deps for the distros in [build_deps.sh](https://github.com/flightlessmango/MangoHud/blob/master/build_deps.sh)